a10_octavia/controller/worker/tasks/a10_database_tasks.py

```diff
@@ -302,6 +308,19 @@ def execute(self, loadbalancer, vthunder_config):
                       loadbalancer.id)
             raise e
 
+    def revert(self, result, loadbalancer, vthunder_config, *args, **kwargs):
+        if isinstance(result, failure.Failure):
```

Comment:
This part seems to resolving the issue related to when revert happens from any other task and vthunder object is not. Maybe its not even needed. But for now keeping it for this PR.